javascript/javascript1/week1/housey_pricey.js

```diff
@@ -0,0 +1,30 @@
+//Peter`s case:
+let width = 8;
+let depth = 10;
+let heght = 10;
+let gardenSizeInM2 = 100;
+let paidPrice = 2_500_000;
+
+//First we calculte house volume:
+let volumeInMeters = depth * heght * width;
+// then use it in house price formula:
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+
+console.log(`Peter's house price should be ${housePrice}, and he paid ${paidPrice}`);
+
```

Comment:
Minor 'extra' assignment: Can you also log whether the person, should actually buy the house?

eg. 'Peter should buy the house: @{shouldBuyTheHouse}'

Reply:
Added extra boolean to show 'Peter should buy the house: @{shouldBuyTheHouse}'